doc/website/blog/2023-10-19-atlas-v-0-15.mdx

```diff
@@ -0,0 +1,459 @@
+  using the `schema apply` command. In this release we have added a new "Lint and Edit" mode to this command, which
+  will analyze your schema changes for issues and allow you to edit them interactively before applying them to your
+  database.
+* **Functions and Stored Procedures** - Atlas now supports creating and managing functions and stored procedures in
+  your database schema.
+* **Postgres Domains** - In addition, Atlas now supports [Postgres Domains](https://www.postgresql.org/docs/16/sql-createdomain.html)
+  as well. A domain is essentially a data type with optional constraints (restrictions on the allowed set of values).
+* **TypeORM Integration** - TypeORM is a popular ORM for Node.js. In this release, we are happy to announce the TypeORM
+  integration, which allows you to automatically generate your database schema from your TypeORM entities, create visualizations
+  and more.
+
+Let's dive right in!
+
+### Interactive Declarative Migrations
+
+Atlas supports a Terraform-like workflow for managing your database schema using the `schema apply` command.
+This workflow, which we call "Declarative Migrations", is a modern alternative to the traditional "versioned migrations"
+workflow. In declarative migrations, you define your desired schema in one of the formats supported by Atlas and supply
+a connection string to your database. Atlas compares the current and desired schema of your database and generates a
+migration plan to bring your database to the desired state.
+
+Similar to Terraform, until today, Atlas would prompt you to confirm the migration plan before applying it to your
+database. This is a great way to ensure that you don't accidentally apply a migration that you didn't intend to. However,
+this flow suffers from a few drawbacks:
+
+1. *Ensuring Safety* - you can count on Atlas to generate a *correct* migration plan to your desired state,
+   but it's still possible that this migration will have unintended side effects. For example, adding a `UNIQUE`
+   constraint to a column might fail if there are duplicate values in the column.
+2. *Editing* - users often want to make changes to their migration plan before applying it. In the current flow, this
+   requires running `schema apply` with the `--dry-run` flag, saving the output to a file, editing it, and then manually
+   applying the edited migration plan to the database.
+
+#### Enter: Interactive Declarative Migrations
+
+In this release, we are introducing a new "Lint and Edit" mode to the `schema apply` command. This mode is
+available to logged-in users only, as it uses Atlas Cloud to provide a neat UI and rich analysis capabilities.
+Let's see it in action.
+
+Start by downloading the latest version of Atlas:
+
+```
+curl -sSf https://atlasgo.sh | sh
+```
+For installation instructions on other platforms, see the [installation guide](https://atlasgo.io/getting-started#installation).
```

Comment:
```suggestion
For more in-depth installation instructions, see the [installation guide](https://atlasgo.io/getting-started#installation).
```

Reply:
The point is to let people know that other platforms exist